docs/auth.md

```diff
@@ -1,11 +1,11 @@
 ---
-id: authentification
-title: "Authentification"
+id: authentication
```

Comment:
Change here you would need to  update `website/sidebars.json` links which uses the wrong id `authentification`.